test/data/car.cpp.h

```diff
@@ -0,0 +1,312 @@
+/**
+ * \file Vehicle.h
+ */
+
+#ifndef Vehicle_h
+#define Vehicle_h
+
+
+
+
+class Vehicle {
+  private:
```

Comment:
I agree, I choose this way of doing it because `hasProtectedFields` and `hasPublicFields` were not giving the expected result in the template. However, `hasPrivateFields` work fine.

Reply:
ok. I could check this out at some point but I’m fine with this for now.